vendor/github.com/containers/image/docker/archive/dest.go

```diff
@@ -19,15 +19,31 @@ func newImageDestination(ctx *types.SystemContext, ref archiveReference) (types.
 	if ref.destinationRef == nil {
```

Comment:
@umohnani8 Change your title to [WIP] until code gets vendored in, for any code that required vendoring.